concent_api/core/queue_operations.py

```diff
@@ -1,2 +1,27 @@
-def send_blender_verification_request():  # TODO implement sending real verivication_request to Work Queue
+from conductor.tasks import blender_verification_request
+from utils.helpers import get_source_package_path
+from utils.helpers import get_storage_file_path
+
+
+def send_blender_verification_request(compute_task_def):
+    task_id = compute_task_def['task_id']
+    subtask_id = compute_task_def['subtask_id']
+    source_package_path = get_source_package_path(subtask_id, task_id)
+    result_package_path = get_storage_file_path(subtask_id, task_id)
+    output_format = get_output_format_from_package(result_package_path)  # pylint: disable=assignment-from-no-return
+    scene_file = get_scene_file_from_package(result_package_path)  # pylint: disable=assignment-from-no-return
```

Comment:
Please remove those `pylint: disable`s. Avoid disabling warnings unless absolutely necessary.

In this case pylint is 100% right. Your functions do not return anything. If the code is unfinished it's a good thing that it has warnings indicating that. Why would we want to hide this information?